src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -633,6 +624,24 @@ def process_declarations(self, parent, nodes, arg_list):
                                 "Could not process {0}. Unrecognized "
                                 "attribute '{1}'.".format(decl.items,
                                                           str(attr)))
+                    elif isinstance(attr, Fortran2003.Intent_Attr_Spec):
```

Comment:
I get the following pycodestyle error (in a part of the code that you have not modified). If you get it as well could you please fix. Otherwise please ignore.
611:73: E502 the backslash is redundant between brackets

Reply:
I also get the error. I removed the redundant backslash now.